### PR TITLE
Dev icons draw

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

**Comment by pfanguin** [comment]
Test link : https://mf-geoadmin3.dev.bgdi.ch/dev_icons_draw

File: src/components/draw/partials/draw.html [hunk 1]
```diff
@@ -99,11 +99,12 @@
...
         <div class="ga-select-box ga-select-icons ga-item-size-{{options.iconSize.value[0]}}"
              ng-class="{'ga-select-box-open': gaSelectBoxOpen}">
-          <img tabindex=1 draggable="false" 
+         <i tabindex=1 draggable="false" ng-repeat="i in options.icons" class="ga-icon-draw icon-{{i.class}}" ng-click="options.icon = i;$event.preventDefault();"></i>
+         <!-- <img tabindex=1 draggable="false" 
                ng-repeat="i in options.icons" 
                ng-class="{'ga-selected': (options.icon == i)}" 
                ng-src="{{i.url}}"
-               ng-click="options.icon = i;$event.preventDefault();"/> 
+               ng-click="options.icon = i;$event.preventDefault();"/> -->
```
Comment (oterral) [inline src/components/draw/partials/draw.html]: Remove the comment

File: src/components/draw/style/draw.less [hunk 2]
```diff
@@ -289,6 +289,11 @@
          border-radius: 10px; 
       }
     }
+    
+    .ga-icon-draw {
+      font-size: 32px;
+      color: #ff0000;
+    }
```
Comment (oterral) [inline src/components/draw/style/draw.less]: Add `text-align:center`

File: src/js/DrawController.js [hunk 3]
```diff
@@ -228,9 +228,11 @@ goog.provide('ga_draw_controller');
         for (var i = 0, ii = $scope.options.icons.length; i < ii; i++) {
           var icon = $scope.options.icons[i];
           icon.url = gaGlobalOptions.resourceUrl + 'img/maki/' + icon.id + '-24@2x.png';
+          icon.class = icon.id;
         }
         $scope.getIconUrl = function(i) {
           return i.url;
+          return i.class;
```
Comment (oterral) [inline src/js/DrawController.js]: This line is useless.

**Comment by oterral** [comment]
Works really good. Is it possible to a namespace to the maki icon ? I want to be sure we don't forget that these icons are not from fontawesome.
We have already `ìcon-ga-XXX` for our custom icon may be an `ìcon-maki-XXX` prefix will be nice

**Comment by pfanguin** [comment]
It's done

File: src/js/DrawController.js [hunk 4]
```diff
@@ -228,6 +228,7 @@ goog.provide('ga_draw_controller');
         for (var i = 0, ii = $scope.options.icons.length; i < ii; i++) {
           var icon = $scope.options.icons[i];
           icon.url = gaGlobalOptions.resourceUrl + 'img/maki/' + icon.id + '-24@2x.png';
+          icon.maki = icon.id;
```
Comment (oterral) [inline src/js/DrawController.js]: ` icon-maki` seems useless. Use directly `icon.id` in the partial html

File: src/components/draw/partials/draw.html [hunk 5]
```diff
@@ -99,14 +99,10 @@
         <span translate>{{'modify_icon_label'}}:</span> 
         <div class="ga-select-box ga-select-icons ga-item-size-{{options.iconSize.value[0]}}"
              ng-class="{'ga-select-box-open': gaSelectBoxOpen}">
-          <img tabindex=1 draggable="false" 
-               ng-repeat="i in options.icons" 
-               ng-class="{'ga-selected': (options.icon == i)}" 
-               ng-src="{{i.url}}"
-               ng-click="options.icon = i;$event.preventDefault();"/> 
-        <button class="ga-icon icon-caret-down"
+          <i tabindex=1 draggable="false" ng-repeat="i in options.icons" class="ga-icon-draw icon-maki-{{i.maki}}" ng-click="options.icon = i;$event.preventDefault();"></i>
```
Comment (oterral) [inline src/components/draw/partials/draw.html]: replace `i.maki` par ` i.id`

Comment (oterral) [inline src/components/draw/partials/draw.html]: and remove `draggable=false`

File: src/js/DrawController.js [hunk 6]
```diff
@@ -228,6 +228,7 @@ goog.provide('ga_draw_controller');
         for (var i = 0, ii = $scope.options.icons.length; i < ii; i++) {
           var icon = $scope.options.icons[i];
           icon.url = gaGlobalOptions.resourceUrl + 'img/maki/' + icon.id + '-24@2x.png';
+          icon.id;
```
Comment (oterral) [inline src/js/DrawController.js]: hmmm what is it used for?

**Pull request merged**